integration/kubernetes/overhead/run-all.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+readonly script_dir=$(dirname $(readlink -f "$0"))
+run_sh="${script_dir}/run.sh"
+timeout=10m
```

Comment:
Why `10m`?

Reply:
arbitrary number were I expect that a test does not take more than that, I will add a comment.
The main reason is actually to avoid that the CI runs for ever, I dont want to limit a lot because each test may take different time.

Let see if we can come with a better number:

In the workloads here is do:
- create a a pod (should be good to say I take 1minute?) 1min
- wait for the pod client  server can communicate: the ntttcp is the one that takes longer the first connection delay for 2 minute : This is due to the first connection never works.
- get overhead samples of 5 seconds : just 5 secounds
- destroy the pods : 30 seconds?

Probably we can set a max time to finish a test  of 4 minutes